package.json

```diff
@@ -47,7 +47,7 @@
     "react-qrcode-logo": "^2.9.0",
     "react-redux": "^8.0.5",
     "react-router-dom": "^5.3.4",
-    "signify-ts": "github:WebOfTrust/signify-ts#faeebad4bdb29832cb25b83e41db064ede07dd36",
+    "signify-ts": "github:cardano-foundation/signify-ts#c1422b041d2ff53b3d4456a29443722b95f35fb5",
```

Comment:
Can you also update the KERIA version in `services/docker-compose.yml` so I know what you have tested against